docker/nginx/conf.d/server/server.api

```diff
@@ -21,6 +21,7 @@ client_max_body_size 128k;
 rewrite ^/portals /skynet/portals permanent;
 rewrite ^/stats /skynet/stats permanent;
 rewrite ^/skynet/blacklist /skynet/blocklist permanent;
+rewrite ^/docs(?:/(.*))?$ https://sdk.skynetlabs.com/$1 permanent;
```

Comment:
So this is redirecting `/docs` right? And we are keeping that endpoint just for backwards-compatibility? I feel like we don't need it if we are moving off of `siasky.net` as we don't have to worry about existing `skynetlabs.com/docs` links.

Reply:
we don't need skynetlabs.com to be backwards compatible with /docs because no one ever used /docs on skynetlabs.com yet

we are keeping someportal.com/docs redirect to sdk.skynetlabs.com as a backwards compatibility for a while because some people might have bookmarked it and we want them redirected